src/test/java/Pom_StepDefinitions/RegisterAndLoginTest.java

```diff
@@ -0,0 +1,5 @@
+package Pom_StepDefinitions;
+
+public class RegisterAndLoginTest {
+
```

Comment:
Hi Prabhat,

Please add step def. file for the feature file.

Thanks,
Prabhat.

Reply:
Added step definition file as per Prabhat Comments.